src/yb/master/master_main.cc

```diff
@@ -69,6 +71,16 @@ static int MasterMain(int argc, char** argv) {
   // Reset some default values before parsing gflags.
   FLAGS_rpc_bind_addresses = strings::Substitute("0.0.0.0:$0", kMasterDefaultPort);
   FLAGS_webserver_port = kMasterDefaultWebPort;
+
+
+  auto host_name = GetHostname();
```

Comment:
minor: it might be cleaner to write this as `Status s = GetHostname(&hostname); if (s.ok())` or at least write the type as `Result<String> otherwise the *host_name access feels a bit cryptic below.